pkg/securitycontext/util.go

```diff
@@ -124,6 +124,20 @@ func DetermineEffectiveSecurityContext(pod *v1.Pod, container *v1.Container) *v1
 	return effectiveSc
 }
 
+// DetermineEffectiveRunAsUser returns a UID from the provided pod's and
+// container's security context. Container's runAsUser take precedence in cases
+// where both are set.
+func DetermineEffectiveRunAsUser(pod *v1.Pod, container *v1.Container) *int64 {
+	runAsUser := new(int64)
```

Comment:
This method needs unit tests. I expected this to return nil if neither the pod nor container specified a runAsUser, but this returns &0 in that case.

Reply:
done